Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/ui/validationresult/common/listitem/AffectedFieldsMapper.kt

```diff
@@ -86,7 +86,7 @@ private fun certificateValue(field: String, certificate: CwaCovidCertificate): S
         }
 
         is TestCertificate -> when (field) {
-            "t.0.tg" -> certificate.targetName
+            "t.0.tg" -> certificate.targetDisease
```

Comment:
Why did this property change?

Reply:
all certificates have `tg` it was called for TC differently for consistency I renamed here , have a look at VC , RC?